src/coordinatorapi.cpp

```diff
@@ -386,16 +386,26 @@ string CoordinatorAPI::getCoordCkptDir(void)
   // FIXME: Add a test for make-check.
   char buf[PATH_MAX] = {0};
   if (noCoordinator()) return "";
-  DmtcpMessage msg(DMT_GET_CKPT_DIR);
-  _coordinatorSocket << msg;
+  jalib::JSocket sock = createNewSocketToCoordinator(COORD_ANY);
+  JASSERT(sock.isValid());
+  DmtcpMessage msg(DMT_GET_COORD_CKPT_DIR);
+  sock << msg;
 
   msg.poison();
-  _coordinatorSocket >> msg;
+  sock >> msg;
   msg.assertValid();
-  JASSERT(msg.type == DMT_GET_CKPT_DIR_RESULT) (msg.type);
+  JASSERT(msg.type == DMT_GET_COORD_CKPT_DIR_RESULT) (msg.type);
+
+  if (msg.extraBytes > 0) {
+    sock.readAll(buf, msg.extraBytes);
+  }
+  else {
+    JWARNING("Failed to get the coordinator checkpoint dir. "
```

Comment:
`JWARNING()` has the same syntax as `JASSERT()`. Please change this to:
`JWARNING(false).Text("...")`